Core/FileLoader.swift

```diff
@@ -27,15 +27,22 @@ enum FileError: Error {
 
 class FileLoader {
 
-    func load(name: String, ext: String) throws -> Data {
+    func load(fileName: String) throws -> Data {
```

Comment:
This function isn't used, and is a little unclear about where files are loaded from as it uses the bundle this class was loaded from (i.e. Core). Delete it?

Reply:
It is used by the TermsOfServiceStore to load tosdr.json. I can rename is BundleFileLoader and make the bundle an explicit argument

Reply:
`TermsOfServiceStore` uses the one that takes a bundle as a parameter.

Reply:
Good point! 🔥